seqr/utils/elasticsearch/es_search.py

```diff
@@ -1274,8 +1274,8 @@ def _annotations_filter(annotations):
 
 
 def _dataset_type_for_annotations(annotations):
-    sv = bool(annotations.get('structural'))
-    non_sv = any(v for k, v in annotations.items() if k != 'structural')
+    sv = bool(annotations.get('structural')) or bool(annotations.get('structural consequence'))
```

Comment:
these need an underscore, no? `'structural_consequence'` not `'structural consequence'`

Reply:
You are right. I changed the front end from `'structural consequence'` to `'structural_consequence'`. It didn't change until I re-ran `npm start`.